ERCS/erc-5559.md

```diff
@@ -1,471 +1,449 @@
 ---
 eip: 5559
-title: "Cross Chain Write Deferral Protocol"
-description: The cross chain write deferral protocol provides a mechanism to defer the storage & resolution of mutations to off-chain handlers
-author: Paul Gauvreau (@0xpaulio), Nick Johnson (@arachnid)
-discussions-to: https://ethereum-magicians.org/t/eip-cross-chain-write-deferral-protocol/10576
-status: Stagnant
+title: "Cross-chain Write Deferral Protocol"
+description: The cross-chain write deferral protocol provides a mechanism to replace L1 storage with L2 and databases through off-chain handlers
+author: Avneet Singh (@sshmatrix), 0xc0de4c0ffee (@0xc0de4c0ffee), Nick Johnson (@arachnid), Makoto Inoue (@makoto), Paul Gauvreau (@0xpaulio)
+discussions-to: https://ethereum-magicians.org/t/eip-5559-cross-chain-write-deferral-protocol/19664
+status: Draft
 type: Standards Track
 category: ERC
-created: 2022-06-23
-requires: 712
+created: 2024-04-15
```

Comment:
The original creation date should stay as is

Reply:
Why? Is that the standard practise?

Reply:
@Arachnid can probably comment on it but I would be very strange that creation date changes as if all the prior specification didn't exist.

Reply:
@SamWilsn & @abcoathup can probably clear this up ↑